### PR TITLE
Remove PipelineRunStatus from deprecation.md

### PR DESCRIPTION
**Pull request by JeromeJu**
# Changes
This commit removes PipelineRunStatus from deprecation.md.

/kind documentation

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [x] Has [Docs](https://github.com/tektoncd/community/blob/main/standards.md#docs) included if any changes are user facing
- [n/a] Has [Tests](https://github.com/tektoncd/community/blob/main/standards.md#tests) included if any functionality added or changed
- [x] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
- [x] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including
  functionality, content, code)
- [x] Has a kind label. You can add one by adding a comment on this PR that contains `/kind <type>`. Valid types are bug, cleanup, design, documentation, feature, flake, misc, question, tep
- [x] Release notes block below has been updated with any user facing changes (API changes, bug fixes, changes requiring upgrade notices or deprecation warnings)
- [x] Release notes contains the string "action required" if the change requires additional action from users switching to the new release

# Release Notes

```release-note
NONE
```

### CODE REVIEW

**Review by jerop — APPROVED** [review]
@JeromeJu please add more details in the commit message, such as a reference to the PRs where the fields and feature flag were removed

File: docs/deprecations.md [hunk 1]
```diff
@@ -19,8 +19,7 @@ being deprecated.
...
-| [The `PipelineRun.Status.TaskRuns` and `PipelineRun.Status.Runs` fields; the `full` and `both` `embedded-status` values along with their functionalities are deprecated and will be removed in v0.45.](https://github.com/tektoncd/community/blob/main/teps/0100-embedded-taskruns-and-runs-status-in-pipelineruns.md) | v0.35.0    | Beta                                                                   | Jan 25, 2023                        |
+| [`PipelineResources` are deprecated.](https://github.com/tektoncd/community/blob/main/teps/0074-deprecate-pipelineresources.md)                                                                                              | [v0.30.0](https://github.com/tektoncd/pipeline/releases/tag/v0.30.0) | Alpha                                                                                                  | Dec 20 2021                         |
```
Comment (lbernick) [inline docs/deprecations.md:22]: why change this line?

**Review by lbernick — COMMENTED** [review]
could you note in the commit message that these fields have already been removed?

File: docs/deprecations.md (hunk 1 above)
Comment (JeromeJu) [reply docs/deprecations.md:22]: Sorry, restored, did not to mean to bring in this PR

**Comment by lbernick** [comment]
> @JeromeJu please add more details in the commit message, such as a reference to the PRs where the fields and feature flag were removed

+1, can you please update the PR description to reference the PRs where the fields were removed?

/lgtm
(description can be edited after the PR is merged)

**Comment by JeromeJu** [comment]
> > @JeromeJu please add more details in the commit message, such as a reference to the PRs where the fields and feature flag were removed
> 
> +1, can you please update the PR description to reference the PRs where the fields were removed?
> 
> /lgtm (description can be edited after the PR is merged)

Thanks Lee! Done

**Pull request merged**